src/pdfdc29.inc

```diff
@@ -921,7 +921,30 @@ wxPdfDCImpl::DoDrawRotatedText(const wxString& text, wxCoord x, wxCoord y, doubl
   CalculateFontMetrics(&desc, fontToUse->GetPointSize(), &height, NULL, &descent, NULL);
   y += (height - abs(descent));
 
-  m_pdfDocument->SetTextColour(m_textForegroundColour.Red(), m_textForegroundColour.Green(), m_textForegroundColour.Blue());
+  if (m_cachedColourSet)
```

Comment:
I think this block of code could be simplified as:
```
if (!m_cachedColourSet || m_textForegroundColour.GetRGB() != m_cachedColourRGB)
{
    m_cachedColourSet = true;
    m_cachedPdfColour = m_textForeground;
}
```
and maybe even more if we remove `m_cachedColourSet` as proposed above.

We definitely shouldn't repeat the same 4 lines of code setting the cached colour twice as is done currently.